libbeat/processors/kubernetes/_meta/fields.yml

```diff
@@ -0,0 +1,29 @@
+- key: kubernetes
+  title: Kubernetes info
+  description: >
+    Kubernetes metadata added by the kubernetes processor
+  fields:
+    - name: kubernetes.pod.name
+      type: keyword
+      description: >
+        Kubernetes pod name
+
+    - name: kubernetes.namespace
+      type: keyword
+      description: >
+        Kubernetes namespace
+
+    - name: kubernetes.labels
+      type: object
+      description: >
+        Kubernetes labels map
+
+    - name: kubernetes.annotations
```

Comment:
This shoud be singular. Reason if someone is going to "read it" it will be `kubernetes.label.testlabel=18`